esp-wifi/Cargo.toml

```diff
@@ -13,6 +13,7 @@ categories = ["embedded", "hardware-support", "no-std"]
 
 [dependencies]
 defmt = { version = "0.3.8", optional = true }
+bleps = { git = "https://github.com/bjoernQ/bleps", package = "bleps", rev = "a5148d8ae679e021b78f53fd33afb8bb35d0b62e", features = [ "macros", "async"] }
```

Comment:
that's not good - we won't be able to release esp-wifi with a git dependency - and also users are free to use any BLE stack they want